patches/tModLoader/Terraria/ModLoader/ChestLoot.Entry.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System;
+using Terraria.ID;
+
+namespace Terraria.ModLoader;
+
+public partial class ChestLoot {
+	private static Item EmptyInstance => new();
+
+	public class Entry {
```

Comment:
With `Entry` being public, we need a review of the public API surface, to make sure that all `public` functions are meant to be used by modders, and that there aren't duplicates with the main class.